conda-forge.yml

```diff
@@ -12,3 +12,5 @@ provider:
   linux_aarch64: default
   linux_ppc64le: default
 test: native_and_emulated
+azure:
+  store_build_artifacts: true
```

Comment:
fair enough to test with this, but let's remove this & rerender before we merge to main